server/gokbg3/grails-app/init/gokbg3/BootStrap.groovy

```diff
@@ -565,6 +563,8 @@ class BootStrap {
     RefdataCategory.lookupOrCreate("Tipp.LinkType", "Parent").save(flush: true, failOnError: true)
     RefdataCategory.lookupOrCreate("Tipp.LinkType", "Previous").save(flush: true, failOnError: true)
 
+    ['Monograph', 'Serial', 'Database', 'Other'].each { type -> RefdataCategory.lookupOrCreate("Tipp.PublicationType", type).save(flush: true, failOnError: true) }
```

Comment:
The default category name should be the classes' `simpleName` `TitleInstancePackagePlatform`. Those categories above are currently unused but should probably adjusted accordingly in any case.

Reply:
you mentioned that before. so: the easiest solution will be to replicate the TitleInstance.Medium as TitleInstancePackagePlatform.Medium and keep both lists in sync.